docs/standard/serialization/add-element-for-schemaimporterextensions.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: "&lt;add&gt; Element for &lt;xmlSchemaImporterExtensions&gt;"
+title: "&lt;add&gt; Element for &lt;schemaImporterExtensions&gt;"
```

Comment:
@louistio as the file is renamed, please update the json file with redirections:
https://github.com/dotnet/docs/blob/master/.openpublishing.redirection.json

Things to remember:
- `source_path` ends with the file extension ".md"
- `redirect_url` starts with `/`, no file extension; this bit is appended to the docs.microsoft.com